docs/percentile_approximation.md

```diff
@@ -22,23 +22,26 @@ CREATE TABLE response_times (
 );
 -- and we'll make it a hypertable for ease of use in the rest of the example
 SELECT create_hypertable('response_times', 'ts');
+-- utilities for generating random numbers
+CREATE SEQUENCE rand START 567;
+CREATE FUNCTION test_random() RETURNS float AS
+    'SELECT ((nextval(''rand'')*34567)%1000)::float/1000'
+LANGUAGE SQL;
```

Comment:
Can we have doctester define this for us?

Reply:
I updated doctester to have the startup script be a part of doctester itself.,